packages/jhupllib/jhupllib.0.1.1/url

```diff
@@ -1,3 +1,3 @@
 http:
   "https://github.com/JHU-PL-Lab/jhu-pl-lib/archive/824006ad7c0b05d015c3449f168bb5e99882bee5.zip"
-checksum: "4e5d5216bc467bdfbf561966d4a748b3"
+checksum: "88cf95af4ba9053d0873577953f714be"
```

Comment:
My `md5sum` for that archive is `3996f5c6a55b8bd37db7c3db11724641`. How did you get that value?

Also, `http:` as field is deprecated, please use `archive:`